dataprofiler/tests/data_readers/test_csv_data.py

```diff
@@ -208,6 +208,31 @@ def test_specifying_data_type(self):
                              input_file['delimiter'],
                              input_file["path"])
 
+    def test_specifying_data_type_as_streams(self):
+        """
+        Determine if the csv file can be loaded with manual data_type setting
+        for streams
+        """
+        for input_file in self.input_file_names:
+            with open(input_file['path'], 'rb') as fp:
+                byte_string = BytesIO(fp.read())
+                input_data_obj = Data(byte_string, data_type='csv')
+                self.assertEqual(input_data_obj.data_type, 'csv')
+                self.assertEqual(input_data_obj.data_type, 'csv', input_file["path"])
+                self.assertEqual(input_data_obj.delimiter,
+                                input_file['delimiter'],
+                                input_file["path"])
+
+            with open(input_file['path'], 'r', encoding=input_file['encoding']) as fp:
+                byte_string = StringIO(fp.read())
```

Comment:
not a blocker for approval, but `byte_string` maybe should be `buffer` or `stream`